src/stan-users-guide/finite-mixtures.Rmd

```diff
@@ -162,7 +162,7 @@ The posterior $p(z_n \mid y_n, \mu, \sigma)$ over the mixture indicator $z_n
 posterior probability that that observation $y_n$ was generated by
 mixture component $k$.  The posterior can be computed via Bayes's rule,
 \begin{align*}
-\operatorname{Pr}\left(z_n = k \mid y_n, \mu, \sigma, \lambda \right)
+\Pr\left(z_n = k \mid y_n, \mu, \sigma, \lambda \right)
```

Comment:
'(' to '[' and similarly for closing parens.  If you are using `\left[`, then it should have a bit of negative space, `\Pr\!\left[` in order to not leave too much space between the "Pr" and the "[".   The alternative is to make \Pr take an argument and put it in the brackets, i.e.,

```latex
\newcommand{\Pr}[1]{\textrm{Pr}\!\left[ #1 \right]}
```